src/hooks/screen/useChatScreenHook.ts

```diff
@@ -219,20 +218,20 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
 
       let response;
       if (type === 'ANONYMOUS') {
-        response = await AnonymousMessageRepo.sendAnonymousMessage(
-          selectedChannel?.id,
+        response = await sendChatAnonMutation.mutateAsync({
+          channelId: selectedChannel?.id,
           message,
-          newAttachments,
-          null
-        );
+          attachments: newAttachments,
+          replyMessageId: null
+        } as any);
       } else {
-        response = await SignedMessageRepo.sendSignedMessage(
-          selectedChannel?.id,
+        response = await sendChatSignedMutation.mutateAsync({
+          channelId: selectedChannel?.id,
           message,
           channelType,
-          newAttachments,
-          null
-        );
+          attachments: newAttachments,
+          replyMessageId: null
+        } as any);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The calls to `AnonymousMessageRepo.sendAnonymousMessage` and `SignedMessageRepo.sendSignedMessage` have been replaced with calls to `sendChatAnonMutation.mutateAsync` and `sendChatSignedMutation.mutateAsync` respectively. These mutation functions were created using the `useSendAnonMessage` and `useSendSignedMessage` hooks.
```diff
- response = await AnonymousMessageRepo.sendAnonymousMessage(
+ response = await sendChatAnonMutation.mutateAsync({
- response = await SignedMessageRepo.sendSignedMessage(
+ response = await sendChatSignedMutation.mutateAsync({
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->